src/Domain/Entities/Servers/Enums/ServerIdentifier.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Crpg.Domain.Entities.Servers.Enums;
+public enum ServerIdentifier
+{
+    // <Information> cRPG Servers are going up to 4, there are letters for each <Information>
+    // <Information> a is battle server <Information>
+    // <Information> b is siege '' <Information>
+    // <Information> c is duel '' <Information>
+    // <Information> d is skirmish '' <Information>
+    // <Information> e is DTV '' <Information>
```

Comment:
What do you think of this?
// 01=Eu 02=NA  03=AS 04=OCE
// A=Battle B=Siege C=Duel D=Skirmish E=DTV

Lightweight, conveys all the information needed.
Also i do not inderstand what does <Information> do? And if it's XML why does it not end with </Information> ?